### ISSUE TITLE
onDrag event not being fired with experimental_preventWindowScrollY on stock browser on android physical device 

### ISSUE THREAD

**Issue by PaoloDiBello** [pmndrs/react-use-gesture#267]
**Describe the bug**
> The drag event is not triggered on the stock browser on an Android phone.

**Sandbox**
>   https://codesandbox.io/s/festive-mirzakhani-21ucz?file=/src/index.js:873-938 

> Use the useDrag hook.
> Use the following options  

   ```
{
      axis: 'y',
      experimental_preventWindowScrollY: true,
      filterTaps: true, // prevent click when dragging
    }
```

**Video**
> https://streamable.com/p05zif

**Information:**
 - React Use Gesture version: ^9.0.4
 - Device: Xiaomi Redmi Note 9 Pro
 - OS: Android 10
 - Browser: stock browser

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [ ] I've tried setting `touch-action: none` to the draggable element. 
     PreventScrollY essentially preventDefault on scroll when a long press is detected
     So I don’t need touch-action
  - [x] Tried with Chrome and I don't have the same issue .
  - [x] Tried with domTarget

**Comment by dbismut** [comment]
Hi @PaoloDiBello I've just checked on my test android phone and Chrome, Samsung, Edge, Firefox. I installed Xiami browser Mi Browser, and it doesn't work indeed (but no drag interactions work at all so I suspect this must have something to do with pointer events). I'll see what I can do for v10.

**Comment by dbismut** [comment]
Fixed in 9.1
